cranelift/codegen/src/isa/aarch64/abi.rs

```diff
@@ -13,16 +13,15 @@ use alloc::boxed::Box;
 use alloc::vec::Vec;
 use regalloc::{RealReg, Reg, RegClass, Set, Writable};
 use smallvec::SmallVec;
-use std::convert::TryFrom;
 
 // We use a generic implementation that factors out AArch64 and x64 ABI commonalities, because
 // these ABIs are very similar.
 
 /// Support for the AArch64 ABI from the callee side (within a function body).
-pub type AArch64ABIBody = ABIBodyImpl<AArch64MachineImpl>;
+pub(crate) type AArch64ABIBody = ABIBodyImpl<AArch64MachineImpl>;
```

Comment:
Sorry if this is not the right place/time for this suggestion, but: I am a bit confused when it comes to distinguish `Body` from `Call`, by just looking at the names. The comments really help, for that matter. Alternatively, what do you think of renaming `ABIBodyImpl` to `CalleeABIImpl`, and `ABICallImpl` to `CallerABIImpl`? (I just realize the consecutive `i` make it a bit hard to decipher, with any casing... so maybe not such a great idea.)

Reply:
Ah, yes, I like the callee/caller distinction better than "body" / "call"; it's much more intuitive. I've gone ahead with these names (I think the `...ABIImpl` is fine IMHO.) Thanks!